.github/workflows/deploy_dashoard.yml

```diff
@@ -0,0 +1,28 @@
+name: Deploy Dashboard
+
+on:
+  pull_request:
+
+jobs:
+  deploy-dashboard:
+    runs-on: ubuntu-latest
+    
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - uses: actions/setup-python@v4
+        with:
+          python-version: '3.10' 
+
+      - name: Install CLI 
+        run: python -m pip install git+https://github.com/visivo-io/visivo.git@${{ github.head_ref }} --force-reinstall
+
+      - name: Deploy
+        if: github.event.pull_request.closed_at == null
+        run: cd test-projects/simple-database && visivo run && VISIVO_TOKEN=${{ secrets.VISIVO_TOKEN }} visivo deploy -s ${{ github.head_ref }}
+
+      - name: Archive 
```

Comment:
This is great. So we can probably make this action available to people by just doing the 

```
steps:
      - name: Checkout
        uses: actions/deploy-visivo-pr-stage
```

Reply:
Yea, pulling it out to an action is a great idea.